{{cookiecutter.repo_name}}/app/index.html

```diff
@@ -26,6 +26,12 @@
    </script>
 
     <script src="scripts/vendor.js"></script>
+    {% if cookiecutter.sentryDsn %}
+    <!-- Should be initialized after angular has been initialized-->
+    <script src="https://cdn.ravenjs.com/3.10.0/angular/raven.min.js" crossorigin="anonymous"></script>
+    <!-- This CDN build auto-initializes the Angular plugin. -->
+    <script>Raven.config("{{ cookiecutter.sentryDsn }}").install();</script>
```

Comment:
Shouldn't we also be adding `environment` var support. Like `environment="production"`